Source/ALSV4_CPP/Public/Character/AI/ALS_BTTask_GetRandomLocation.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2020, Doga Can Yanikoglu
+// @author: Jens Bjarne Myhre
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "NavigationSystem.h"
+#include "BehaviorTree/Tasks/BTTask_BlackboardBase.h"
+#include "ALS_BTTask_GetRandomLocation.generated.h"
+
+ //Picks a random location reachable through NavMesh within the Max Distance from the Owning Pawn's current location and assigns it to the specified Blackboard Key
+UCLASS(Category=ALS, meta=(DisplayName = "Get Random Location"))
+class ALSV4_CPP_API UALS_BTTask_GetRandomLocation : public UBTTask_BlackboardBase
+{
+	GENERATED_BODY()
+	
+public:
+	UALS_BTTask_GetRandomLocation();
+
+	//Maximum distance the random location picked may be from pawn
+	UPROPERTY(Category = Navigation, EditAnywhere, meta=(ClampMin = 1))
+	float MaxDistance = 1000;
+
+	//Navigation Query filter used when picking the location, only picks locations reachable using this filter when assigned
```

Comment:
Comments in header files should use `/** */` 

Applicable for other similar header comments in this PR.